web_app.py

```
@@ -1649,99 +1649,110 @@ def chat(self, minister_name: str, message: str) -> Dict[str, Any]:
+                    # #501：叙事抽取落账与读心并行尾随（各自 pending ownership，P5）。
+                    self._spawn_extraction_trail(minister_name, answer_text, chat_turn_id)
+                # #544：非流式折进等待窗——判官以超时封顶后与回话同到；超时则空清单先行。
+                if message_id and answer_text:
+                    self._trail_highlight_judge_after_reply(
+                        answer_text,
+                        message_id=message_id,
+                        chat_turn_id=chat_turn_id,
+                    )
+                    payload["history"] = self.chat_projection(minister_name)
+            except Exception:
+                # drain 在 write_gate 外（与 stream / retry 同序），再短写 fail。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)